std/typecons.d

```diff
@@ -1148,6 +1147,24 @@ unittest
         static assert( is(typeof(tc4 < tm4)));
         static assert( is(typeof(tc4 < tc4)));
     }
+    // Bugzilla 14890
+    static void test14890(inout int[] dummy)
```

Comment:
Or just compiling it is enough, since it must be compiled anyway, and the problem was at compiling stage? Anyway, will look fishy for static analysis, e.g., code coverage.